bin/ch/HostConfigFlagsList.h

```diff
@@ -4,7 +4,10 @@
 //-------------------------------------------------------------------------------------------------------
 
 #ifdef FLAG
-FLAG(BSTR, Serialized,                    "If source is UTF8, deserializes from bytecode file", NULL)
-FLAG(BSTR, GenerateLibraryByteCodeHeader, "Generate bytecode header file from library code", NULL)
+FLAG(BSTR, dbgbaseline,                     "Baseline file to compare debugger output", NULL)
+FLAG(BSTR, DebugLaunch,                     "Create the test debugger and execute test in the debug mode", NULL)
```

Comment:
changed to bool

---
In reply to: [62715892](https://github.com/Microsoft/ChakraCore/pull/926#discussion_r62715892) [](ancestors = 62715892)